src/SSHDebugPS/SSHDebugPS.csproj

```diff
@@ -14,9 +14,13 @@
     <RootNamespace>Microsoft.SSHDebugPS</RootNamespace>
     <AssemblyName>Microsoft.SSHDebugPS</AssemblyName>
     <FileAlignment>512</FileAlignment>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <ProjectTypeGuids>{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <OutputPath>$(MIDefaultOutputPath)</OutputPath>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(IsCoreClr)' == 'true'">
```

Comment:
><PropertyGroup Condition="'$(IsCoreClr)' == 'true'"> [](start = 2, length = 52)

Since this is always true, any reason for the change?

Reply:
I put it in to make it more clear.